data/mt/retired/Mark-Noland-9fe046dc-4a09-4f3a-8054-97718b4dc017.yml

```diff
@@ -3,19 +3,27 @@ name: Mark Noland
 given_name: Mark
 family_name: Noland
 gender: Male
-email: marknolandhd10@gmail.com
+email: Mark.Noland@legmt.gov
 party:
 - name: Republican
 roles:
-- type: lower
+- end_date: '2022-12-31'
+  type: lower
   jurisdiction: ocd-jurisdiction/country:us/state:mt/government
   district: '10'
+- type: upper
```

Comment:
@bfossen-ce this person is starting in the senate on 01/02/2023, do they need a start date?